### ISSUE TITLE
Putting out a call-for-maintainers

### ISSUE THREAD

**Issue by patcon** [benwebber/ansible-minecraft#28]
Hi there @benwebber and @joshuacherry! This is great! Thanks so much for sharing it

I notice that you've gotten busy and haven't had time to triage the issue queue. Any chance you'd appreciate my trying to organize a call-for-maintainers to help get some things merged in this repo? Obviously, you'd have final say on who gets added as a contributor, but I could do a the searching through past issues to find candidates who have contributed in the past and maybe have bandwidth to help :)

Regardless, thanks for your work on this, and any consideration you might be able to offer :)

**Comment by goodrum** [comment]
@benwebber and @joshuacherry I would be willing to assist and maintain this repo.  My family uses this code as part of maintaining our private Minecraft servers for several months now.  I am happy to help with getting Pull Requests merged in as well as dealing with some of the outstanding issues.  This repo is used as part of our overall Terraform+Ansible+AWS solution for building dynamic Minecraft servers.  I have even worked through the requirements around backup, restore, and deploying Forge with mods.

**Comment by patcon** [comment]
Thanks for speaking up @goodrum!

I feel like it could be rad if you felt like sharing back some of the others pieces in complementary roles. We could perhaps make the supporting roles more visible from one another's READMEs :)

**Comment by goodrum** [comment]
I have a secondary role that I am working to package.  It contains most of the code for handling the backups/restores and loading custom worlds.  I leverage S3 buckets to perform the process and have a backup service that runs every time we stop the server.  This way all the backups are performed prior to powering off the machine to ensure we don't lose our state.  Combined with Terraform, I am able to destroy everything (but my EIP) and cut my costs to next to nothing.

I have a fork that I am working to enhance and merge in some of these changes.  I have merged in @nolte PR 27 which expertly handles the new Minecraft URL paths.

**Comment by joshuacherry** [comment]
@patcon and @goodrum ,

I'm happy to have more contributors to this role since I haven't had much time lately to work on this. Only @benwebber has the ability to add members to the @devops-coop/minecraft team, but if he doesn't offer an opinion on your request then I will gladly start adding contributors directly to the repo. 

@benwebber, this is originally your work so I would really appreciate your thoughts before any changes are made.

**Comment by patcon** [comment]
Yeah, would definitely love to hear from @benwebber too, if we could give it a bit more time. Wouldn't feel right to jump in without his consent and hearing any expectations he might have :) Thanks @joshuacherry!

**Comment by ianrandmckenzie** [comment]
@benwebber any chance you'd be willing to update the list of maintainers?

Much <3 from Canada!

**Comment by jamesyoungman** [comment]
It's been eighteen months since this thread started and three months since the last comment on this thread.  As far as I can see, no responses from the repo owner.   Is there a consensus across users of this code about what the best way forward is?    Should we basically switch to something like https://github.com/goodrum/ansible-minecraft/tree/feature/overall_enhancements?

**Comment by patcon** [comment]
cc @joshuacherry :)
